js/repl/ReplOptions.js

```diff
@@ -213,6 +215,15 @@ class ExpandedContainer extends Component<Props, State> {
               />
               File Size
             </label>
+
+            <label className={styles.settingsLabel}>
+            File Name
+              <input
+                type= "text"
+                className={styles.inputFileBoxLeft}
+                size={fileNameBoxSize}
+                value={filename}
```

Comment:
For this controlled component to work properly, it needs to have an `onChange` handler to report the changes happening to it. You can probably follow how I did it https://github.com/babel/website/pull/1676/files#diff-ecb6fb48984cd148bfc45fb63ca3f34aR225